record/mpp.go

```diff
@@ -0,0 +1,87 @@
+package record
+
+import (
+	"io"
+
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/tlv"
+)
+
+// MPPOnionType is the type used in the onion to reference the MPP fields:
+// total_amt and payment_addr.
+const MPPOnionType tlv.Type = 14890322186762188277 // temporary, optional field
```

Comment:
Such number, very large, wow

Reply:
ahh yeah i forgot to push up the change that now use the proper type from the spec